docs/apis-tools/tasklist-api-rest/schemas/requests/task-search-request.mdx

```diff
@@ -178,19 +178,23 @@ TaskSearchRequest - query object to search tasks by provided params.
 
 #### [<code style={{ fontWeight: 'normal' }}>TaskSearchRequest.<b>searchAfter</b></code>](#)<Bullet />`string` <Badge class="secondary" text="list"/>
 
-> Array of values copied from `sortValues` of one of the tasks, query will return page of tasks going directly after this values plus same sort values.
+> Array of values that should be copied from `sortValues` of one of the tasks from the current search results page.
+> It enables the API to return a page of tasks that directly follow the provided values in terms of sorting order.
```

Comment:
```suggestion
> It enables the API to return a page of tasks that directly follow the task identified by the provided values, with respect to the sorting order.
```

Let's please add this to all the 4 options, I believe it makes it clearer

Reply:
updated for the 4 fields